### ISSUE TITLE
Notification not show if has "group" in payload

### ISSUE THREAD

**Issue by pewh** [evollu/react-native-fcm#720]
Before openning an issue
1. Reproduce it with the example project in this repo.
2. If you can't receive iOS notification, make sure you can receive notification using quickstart-ios project provided by Firebase team

When openning an issue, please include following information for better support

1. What version of RN and react-native-fcm are you running? 0.50.1 & 11.0.1
2. What device are you using? (e.g iOS9 emulator, Android 6 device)? Android emulator & device (Jelly Bean)
3. Is your app running in foreground, background or not running? Foreground

I test both remote push notification & presentLocalNotification. For quick test, try `presentLocalNotification` example that mentioned at README 
```
FCM.presentLocalNotification({
            id: "UNIQ_ID_STRING",                               // (optional for instant notification)
            title: "My Notification Title",                     // as FCM payload
            body: "My Notification Message",                    // as FCM payload (required)
            priority: "high",                                   // as FCM payload
            click_action: "ACTION",                             // as FCM payload
            badge: 10,                                          // as FCM payload IOS only, set 0 to clear badges
            number: 10,                                         // Android only
            ticker: "My Notification Ticker",                   // Android only
            auto_cancel: true,                                  // Android only (default true)                            // as FCM payload, you can relace this with custom icon you put in mipmap
            big_text: "Show when notification is expanded",     // Android only
            sub_text: "This is a subText",                      // Android only
            color: "blue",                                       // Android only
            vibrate: 300,                                       // Android only default: 300, no vibration if you pass 0
            group: "asd",                                     // Android only
            lights: true,                                       // Android only, LED blinking (default false)
            show_in_foreground: true,                               // notification when app is in foreground (local & remote)
        });
```

and notification will not show. It shows if I remove `group`. Previously it's working, and I don't know which version is working.

**Comment by evollu** [comment]
nothing related to `group` has changed recently.

**Comment by pewh** [comment]
I can reproduce the issue by using Example from this repo. Just add `group` in `presentLocalNotification` then push notification will not work.

When I trigger presentLocalNotification using `group`, `adb logcat` didn't trace anything.

**Comment by evollu** [comment]
can't reproduce with android 8 emulator, what version of emulator are you using

**Comment by pewh** [comment]
I'm using android jelly bean, tested on real device (4.4.2) & genymotion (4.1.1).
But if I testing on android marshallow, it's work.

Previously I didn't have any issue on FCM group payload on jellybean.

**Comment by evollu** [comment]
fixed in 11.0.2
